packages/core/src/button/button.element.spec.ts

```diff
@@ -280,51 +271,74 @@ describe('button element', () => {
       component.disabled = true;
       component.loadingState = ClrLoadingState.default;
       await componentIsStable(component);
-      expect(component.disabled).toBeTruthy();
-    });
-  });
 
-  describe('Button link', () => {
-    let testLinkElement: HTMLElement;
-    let anchor: HTMLAnchorElement;
-    let anchorButton: HTMLButtonElement;
+      component.loadingState = ClrLoadingState.loading;
+      await componentIsStable(component);
+      expect(component.disabled).toBeTruthy();
 
-    beforeEach(async () => {
-      testLinkElement = await createTestElement(html` <a href="about"><cds-button>About</cds-button></a> `);
+      component.loadingState = ClrLoadingState.success;
+      await componentIsStable(component);
+      expect(component.disabled).toBeTruthy();
 
-      anchor = testLinkElement.querySelector<HTMLAnchorElement>('a');
-      anchorButton = testLinkElement.querySelector<HTMLButtonElement>('cds-button');
+      component.loadingState = ClrLoadingState.default;
+      await componentIsStable(component);
+      expect(component.disabled).toBeTruthy();
     });
 
-    afterEach(() => {
-      removeTestElement(testLinkElement);
+    it('should default to spinner size to "18"', async () => {
+      component.loadingState = ClrLoadingState.loading;
+      await componentIsStable(component);
+      expect(component.shadowRoot.querySelector<any>('cds-progress-circle').size).toBe('18');
     });
 
-    it('should render a link properly', async () => {
-      await componentIsStable(anchorButton);
-      expect(anchor.style.lineHeight).toBe('0');
-      expect(anchor.style.textDecoration).toBe('none');
+    it('should set spinner size to "12" if button size "sm"', async () => {
+      component.loadingState = ClrLoadingState.loading;
+      component.size = 'sm';
+      await componentIsStable(component);
+      expect(component.shadowRoot.querySelector<any>('cds-progress-circle').size).toBe('12');
     });
+  });
+});
 
-    it('should set button to be readonly', async () => {
-      await componentIsStable(anchorButton);
-      expect(anchor.querySelector('cds-button').readonly).toBe(true);
-    });
+describe('Button link', () => {
```

Comment:
Moved this describe since it was nested in the describe above which made it a bit confusing what component it was referencing.